main.go

```diff
@@ -106,29 +106,48 @@ func processes() {
 
 	}
 
+	// Print header for all the entries
+	printColumnHeaders()
+
 	for _, p := range ps {
 		buf := bytes.NewBuffer(nil)
 		pid := strconv.Itoa(p.PID)
 		fmt.Fprint(buf, pad(pid, maxPID))
-		fmt.Fprint(buf, " ")
+		fmt.Fprint(buf, "\t")
 		ppid := strconv.Itoa(p.PPID)
 		fmt.Fprint(buf, pad(ppid, maxPPID))
-		fmt.Fprint(buf, " ")
+		fmt.Fprint(buf, "\t")
 		fmt.Fprint(buf, pad(p.Exec, maxExec))
 		if p.Agent {
 			fmt.Fprint(buf, "*")
 		} else {
-			fmt.Fprint(buf, " ")
+			fmt.Fprint(buf, "\t")
 		}
-		fmt.Fprint(buf, " ")
+		fmt.Fprint(buf, "\t")
 		fmt.Fprint(buf, pad(p.BuildVersion, maxVersion))
-		fmt.Fprint(buf, " ")
+		fmt.Fprint(buf, "\t")
 		fmt.Fprint(buf, p.Path)
 		fmt.Fprintln(buf)
 		buf.WriteTo(os.Stdout)
 	}
 }
 
+func printColumnHeaders() {
+	buf := bytes.NewBuffer(nil)
+	fmt.Fprint(buf, "maxPID")
```

Comment:
Camel case is probably not the best formatting style we can use here.

Reply:
So what according to you should be the formatting style?